code/game/objects/items/stacks/barbed_wire.dm

```diff
@@ -72,14 +72,18 @@
 /obj/item/stack/razorwire/attack_self(mob/user) //use barbed wire to deploy it
 	if(!ishuman(usr))
 		return
-	var/turf/target = get_step(user.loc,user.dir)
+	var/turf/open/target = get_step(user.loc,user.dir)
```

Comment:
it is possible that the turf in front of the user is a wall when they attempt to deploy. This might runtime in that case.

Reply:
gonna quickly drop an istype check

Reply:
Still can runtime, type mismatch